Gemfile

```diff
@@ -41,7 +41,7 @@ gem 'travis', '~>1.7.6'
 gem 'awesome_spawn'
 gem 'default_value_for'
 gem 'more_core_extensions', "~> 2.0.0", :require => 'more_core_extensions/all'
-gem 'rubocop', '>= 0.27.1'
+gem 'rubocop', '>= 0.27.1', :require => false
 gem 'haml-lint', '~> 0.13.0'
```

Comment:
I think this ultimately requires rubocop too (https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/rubocop.rb#L2), so might need the same treatment before we gain anything.